### ISSUE TITLE
Filtering prefixes for address family fails with error in GUI

### ISSUE THREAD

**Issue by Kircheneer** [nautobot/nautobot#2913]
### Environment
* Nautobot version (Docker tag too if applicable): 1.5.1
* Python version: 3.7
* Database platform, version: Postgres, 14
* Middleware(s): -

### Steps to Reproduce
1. Filter prefixes for family in the GUI using the new filter feature

Note: Navigate to `/ipam/prefixes/?family=6&q=` (e.g. https://demo.nautobot.com/ipam/prefixes/?family=6&q=)

### Expected Behavior
1. Prefixes to be filtered correctly

### Observed Behavior

```
Invalid filters were specified:

  family
    Enter a number.
```

![Screenshot 2022-11-28 at 09 58 28](https://user-images.githubusercontent.com/12400533/204235788-5f91d8a6-31c6-4028-9b21-148fb6886f12.png)

**Comment by Kircheneer** [comment]
Same thing for IP addresses and aggregates pointing to `IPAMFilterSetMixin` as a potential problem source.

**Comment by Kircheneer** [comment]
Resolved in 1.5.2b1 - I believe this relates to https://github.com/nautobot/nautobot/issues/2837 and it therefore solved by https://github.com/nautobot/nautobot/pull/2879.
